.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,38 @@
+- 
+- 
+- 
+
+## 변경 이유
+<!-- 왜 이 변경이 필요한지 설명해주세요 -->
+- 
+- 
+- 
+
+
+## 체크리스트
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)